pkg/recommendation/recommender/resource/recommend.go

```diff
@@ -48,6 +48,18 @@ func (rr *ResourceRecommender) PreRecommend(ctx *framework.RecommendationContext
 }
 
 func (rr *ResourceRecommender) makeCpuConfig() *config.Config {
+	cpuHistogramHalfLife, exists := rr.Recommender.Config["cpu-histogram-half-life"]
```

Comment:
Please move it to register.go , we use there to centralized management the configurations.